Python/README.md

```diff
@@ -5,9 +5,9 @@
 
 Required software:
   1. Compilers:
-  2. Linters:
+  2. Linters: [flake8](http://flake8.pycqa.org/), [pyflakes](https://github.com/PyCQA/pyflakes), [mypy](http://mypy-lang.org/) *(experimental)*
```

Comment:
That just to make it visible to others.
What about Flow, I think we may add section: "Type checkers" or place it in "Other tools" section.